tests/test_utils/test_data/test_dataset_loader.py

```diff
@@ -0,0 +1,25 @@
+from os.path import exists
+
+import pytest
+
+import nujo.utils.data.nujo_dir as nujo_dir
+from nujo.utils.data import Dataset
+from nujo.utils.data.dataset_loader import DatasetLoader
+
+
+def test_dataset_loader_download(get_dataset):
+    DatasetLoader('iris', 'csv', True).download()
+
+    assert exists(nujo_dir.HOME_DIR + 'iris.data')
+
+
+def test_dataset_loader_install(get_dataset):
+    dataset = Dataset('iris', 'csv', True, True)
+
+    assert dataset.X.shape == get_dataset[0]
+    assert dataset.y.shape == get_dataset[1]
+
+
+@pytest.fixture
+def get_dataset():
+    return (150, 4), (150, 1)
```

Comment:
I can't see where you use this fixture.

Reply:
Lines 16-20?

Reply:
O yeah, mb, but in this case it's better to hard code the tuples in the test method and remove the fixture altogether

Reply:
ok, I thought it was necessary for pytest